src/mpyl/cli/build.py

```diff
@@ -22,14 +23,30 @@
     RUN_ARTIFACTS_FOLDER,
     RUN_RESULT_FILE_GLOB,
 )
-from ..project import load_project
+from ..project import load_project, Target
 from ..stages.discovery import find_projects
 from ..steps import deploy
+from ..steps.models import ConsoleProperties
 from ..steps.run_properties import construct_run_properties
 from ..utilities.pyaml_env import parse_config
 
 
+@dataclass(frozen=True)
+class Context:
+    target: Target
+    config: dict
+    console: Console
+    run_properties: dict
+
+
 @click.group("build")
+@click.option(
+    "--environment",
+    "-e",
```

Comment:
maybe a bit far fetched, but might this mess up with the docker `-e` when passing this arg to our docker container?

Reply:
Good one! It doesn't though, Docker is smart enough to handle this. If you run
```
docker run image:version foobar -e foo=bar
                         ^^^^^^^^^^^^^^^^^
```
It knows everything after the image name are container parameters 👍